tools/ArduinoCsCompiler/tests/FirmataIlExecutorTests.cs

```diff
@@ -315,7 +315,7 @@ public void LargeStructTest(string methodName, Int32 argument1, Int32 argument2,
         [InlineData(nameof(TestMethods.CastClassTest), 0, 0, 1)]
         public void CastTest(string methodName, Int32 argument1, Int32 argument2, Int32 expected)
         {
-            LoadCodeMethod(methodName, argument1, argument2, expected, new CompilerSettings() { CreateKernelForFlashing = true, UseFlashForKernel = true });
+            LoadCodeMethod(methodName, argument1, argument2, expected, new CompilerSettings() { CreateKernelForFlashing = false, UseFlashForKernel = false });
```

Comment:
are those firmata changes related to this PR?

Reply:
Yes, kind of. The changes added a few extra types to the code set required for some tests, exceeding the allocated flash space. This change reduces the memory usage of the tests so they pass again.